assets/js/components/Work/Tabs/Administrative.jsx

```diff
@@ -34,190 +37,158 @@ const WorkTabsAdministrative = ({ work }) => {
     },
     refetchQueries(mutationResult) {
       return [{ query: GET_WORK, variables: { id } }];
-    }
+    },
   });
 
-  const onSubmit = data => {
+  const onSubmit = (data) => {
     let workUpdateInput = {
       administrativeMetadata: {
         preservationLevel: Number(data.preservationLevel),
-        rightsStatement: data.rightsStatement
+        rightsStatement: data.rightsStatement,
       },
       visibility: data.visibility,
 
-      published: true
+      published: true,
     };
     updateWork({
-      variables: { id, work: workUpdateInput }
+      variables: { id, work: workUpdateInput },
```

Comment:
@katdivyareddy10 Are they updating on `deploy/staging`?

Reply:
@kdid @bmquinn  Mind sanity checking this for us?   Seems like we're getting the right response from the API mutation in the app and in GraphQL Playground, but the data doesn't seem to persist.

Reply:
Ok, I think the cache is not updating.  It wasn't working, but now my data magically appeared.  Looking into it.

Reply:
@katdivyareddy10 This is in `deploy/staging` as well, and is happening (I'm pretty sure) because we're calling 2 update `mutation`s in the `onSubmit()` function.  This is messing with the cache.  I'll create a new issue for it.   Not related to this PR.